scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1190,6 +1217,10 @@ def query_api( self, path, pretty=0 ):
             pretty = '?%s' % pretty
 
         url = self._http_host + path + pretty
+
+        import traceback
+        global_log.info('k8s api get %s, %s' % (url, traceback.format_exc()))
```

Comment:
Ok, should this be removed since we throw the exception above?

Reply:
I'm keeping this since we are removing the exception instead